src/tools/dart-analyze.md

```diff
@@ -30,12 +30,22 @@ add the `--fatal-infos` flag:
 $ dart analyze --fatal-infos
 ```
 
-You can add a directory argument:
+You can add a directory or a single file argument:
+
+```terminal
+$ dart analyze [<DIRECTORY> | <DART FILE>]
```

Comment:
```suggestion
$ dart analyze [<DIRECTORY> | <DART_FILE>]
```

(Per https://developers.google.com/style/placeholders#explaining-placeholders)

Reply:
There's really a section for everything, didn't notice this one. Thanks!